README.md

```diff
@@ -1,6 +1,6 @@
 # Nautobot Secrets Providers
 
-Nautobot Secrets Providers is a plugin for [Nautobot](https://github.com/nautobot/nautobot) 1.2.0 or higher that bundles Secrets Providers for integrating with popular secrets backends. Nautobot 1.2.0 added support for integrating with retrieving secrets from various secrets providers.
+Nautobot Secrets Providers is a plugin for [Nautobot](https://github.com/nautobot/nautobot) 1.4.0 or higher that bundles Secrets Providers for integrating with popular secrets backends. Nautobot 1.4.0 added support for integrating with retrieving secrets from various secrets providers.
```

Comment:
Is this accurate? Pypi goes back to version 1.0.0 which supports nautobot `~1.2.1`

Reply:
The template we use has minimal Nautobot version v1.4.

Reply:
@snaselj @gsnider2195 is suggesting that this message is inaccurate. It's more-so speaking to the historical support of Secrets which was introduced in 1.2. We are moving up the minimum version but that doesn't rewrite history.

Reply:
Fixed